site/content/docs/manifest/lb-web-service.md

```diff
@@ -178,6 +180,12 @@ Scale up or down based on the average CPU your service should maintain.
 <span class="parent-field">count.</span><a id="count-memory-percentage" href="#count-memory-percentage" class="field">`memory_percentage`</a> <span class="type">Integer</span>  
 Scale up or down based on the average memory your service should maintain.  
 
+<span class="parent-field">count.</span><a id="requests" href="#count-requests" class="field">`requests`</a> <span class="type">Integer</span>  
+Scale up or down based on the request count handled per tasks.
+
+<span class="parent-field">count.</span><a id="response_time" href="#count-response_time" class="field">`response_time`</a> <span class="type">Integer</span>  
```

Comment:
Should we update the type to `Duration`?

Reply:
:+1: